File: src/app/gkill/api/req_res/add_re_kyou_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type AddReKyouResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	AddedReKyou *reps.ReKyou `json:"added_re_kyou"`

	AddedKyou *reps.Kyou `json:"added_kyou"`
}

File: src/client/classes/api/req_res/add-re-kyou-response.ts
'use strict'

import { ReKyou } from '@/classes/datas/re-kyou'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddReKyouResponse extends GkillAPIResponse {

    added_rekyou: ReKyou

    added_kyou: Kyou | null

    constructor() {
        super()
        this.added_rekyou = new ReKyou()
        this.added_kyou = null
    }

}


